en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md

```diff
@@ -0,0 +1,122 @@
+# ISO8583 Connector Example
+
+Given below is a sample scenario that demonstrates how to WSO2 ISO8583 connector send an ISO8583 message to the financial networks using WSO2 micro integrator.
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
```

Comment:
```suggestion
The ISO8583 message format is used for financial transactions such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce, etc. transactions.
```

Removed the first line. I think it's not required.

Reply:
Done.